examples/basics/plotting/grid_x_y_viewbox.py

```diff
@@ -0,0 +1,48 @@
+
+
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+
+"""
++---+-------------+
+|   |             |
+| y |   viewbox   |
+|   |             |
++---+-------------+
+| sp|     x       |
```

Comment:
Yes, this was what I meant by saying that Cassowary breaks too easily :/ I'm not a fan of completely black-holing every Cassowary exception though. Is there a saner alternative?